dist/templates/ovn-setup.yaml.j2

```diff
@@ -56,6 +56,13 @@ rules:
   - networkpolicies
   - statefulsets
   verbs: ["get", "list", "watch"]
+- apiGroups:
+  - discovery.k8s.io
+  resources:
+  - endpointslices
+  verbs:
+  - list
```

Comment:
I know we don't need `get`, but might as well add it here.

Reply:
done